handlers/people_test.go

```diff
@@ -258,3 +258,191 @@ func TestDeletePerson(t *testing.T) {
 		mockDb.AssertExpectations(t)
 	})
 }
+
+func TestGetPeopleBySearch(t *testing.T) {
+	mockDb := mocks.NewDatabase(t)
+	pHandler := NewPeopleHandler(mockDb)
+
+	t.Run("should return users that match the search text", func(t *testing.T) {
+		rr := httptest.NewRecorder()
+		handler := http.HandlerFunc(pHandler.GetPeopleBySearch)
+		expectedPeople := []db.Person{
+			{ID: 1, Uuid: "uuid1", OwnerPubKey: "pubkey1", OwnerAlias: "John Doe"},
+			{ID: 2, Uuid: "uuid2", OwnerPubKey: "pubkey2", OwnerAlias: "John Smith"},
+		}
+
+		rctx := chi.NewRouteContext()
+		rctx.URLParams.Add("search", "John")
```

Comment:
@AbdulWahab3181 the search should be in the URL query, not param.

Reply:
@elraphty  Addressed